test.html

```diff
@@ -0,0 +1,11 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>test</title>
+  </head>
+  <body>
+    <h1> another page </h1>
+    <h2> rrsdfadsf <h2>
```

Comment:
Please, close the `<h2>`
```suggestion
    <h2> rrsdfadsf </h2>
```